subprojects/ivy/src/main/java/org/gradle/api/publish/ivy/internal/publisher/IvyDescriptorFileGenerator.java

```diff
@@ -281,6 +286,10 @@ private void writeDependencies(OptionalAttributeXmlWriter xmlWriter) throws IOEx
         xmlWriter.endElement();
     }
 
+    private boolean isDynamicVersion(String version) {
+        return VersionRangeSelector.ALL_RANGE.matcher(version).matches() || version.endsWith("+");
```

Comment:
Good catch on the `+` but that makes me realize that `latest.` is not covered.

Reply:
it is now https://github.com/gradle/gradle/pull/9121/commits/5f146ef5bc159990e133d5ffbf2f81d0953a21a6 :)